src/components/questionnaire/new-questionnaire-builder.tsx

```diff
@@ -381,17 +379,9 @@ const NewQuestionnaireBuilder = () => {
         removeQuestionFromPreview={removeQuestionFromPreview}
         editQuestionInPreview={editQuestionInPreview}
       />
-      <Card sx={{ mt: 2, mb: 2, width: "100%" }}>
-        <Link
-          to={adminMode ? "/admin/questionnaire" : "/questionnaire"}
-          style={{ textDecoration: "none" }}
-        >
-          <Button variant="contained" sx={{ p: 2, width: "100%" }}>
-            <KeyboardReturn sx={{ marginRight: "10px" }} />
-            <Typography>{strings.newQuestionnaireBuilder.back}</Typography>
-          </Button>
-        </Link>
-      </Card>
+      <BackButton 
+        sx={{ mt: 3, marginBottom: 2 }}
+      />
```

Comment:
for code consistency

```suggestion
      <BackButton sx={{ mt: 3, marginBottom: 2 }}/>
```